src/UnscentedKalmanInversion.jl

```diff
@@ -666,6 +666,14 @@ function get_u_mean_final(uki::EnsembleKalmanProcess{FT, IT, Unscented}) where {
     return uki.process.u_mean[end]
 end
 
+"""
+    get_u_cov_final(uki::EnsembleKalmanProcess{FT, IT, Unscented}) where {FT <: AbstractFloat, IT <: Int}
+
+Returns the unconstrained parameter covariance at the final iteration.
+"""
+function get_u_cov_final(uki::EnsembleKalmanProcess{FT, IT, Unscented}) where {FT <: AbstractFloat, IT <: Int}
```

Comment:
Instead of writing the method `get_u_cov_final` for UKI, we should instead write the method `get_u_cov` for uki:
```
function get_u_cov(uki::EnsembleKalmanProcess{FT,IT,Unscented}, iteration::IT) where {FT <: AbstractFloat, IT <: Integer}
    return uki.process.uu_cov[it]
end
```
Then we can use the `get_u_cov_final` method from ekp?

Reply:
Sounds good, I will do the same for get_u_mean_final.